src/contracts/core/DelegationManager.sol

```diff
@@ -763,6 +753,14 @@ contract DelegationManager is Initializable, OwnableUpgradeable, Pausable, Deleg
         }
     }
 
+    function _initializeWithdrawalDelayBlocks(uint256 _withdrawalDelayBlocks) internal {
+        require(
+            _withdrawalDelayBlocks <= MAX_WITHDRAWAL_DELAY_BLOCKS,
+            "DelegationManager._initializeWithdrawalDelayBlocks: _withdrawalDelayBlocks cannot be > MAX_WITHDRAWAL_DELAY_BLOCKS"
+        );
+        withdrawalDelayBlocks = _withdrawalDelayBlocks;
+    }
```

Comment:
would be good to still emit the `WithdrawalDelayBlocksSet` event, I think?
feels like we're not committing to this variable never changing, just that it changing it will _require_ an upgrade now

Reply:
Good call, added the event here https://github.com/Layr-Labs/eigenlayer-contracts/pull/355/commits/810b2a115b80d011a2077000a70369abf11b3464